packages/protocol/contracts/verifiers/RiscZeroVerifier.sol

```diff
@@ -80,7 +79,12 @@ contract RiscZeroVerifier is EssentialContract, IVerifier {
         // journalDigest is the sha256 hash of the hashed public input
         bytes32 journalDigest = sha256(bytes.concat(hash));
 
-        if (!receiptVerifier.verify(seal, imageId, postStateDigest, journalDigest)) {
+        // call risc0 verifier contract
+        (bool success,) = address(receiptVerifier).staticcall(
+            abi.encodeWithSignature("verify(bytes,bytes32,bytes32)", seal, imageId, journalDigest)
```

Comment:
It's recommended to use `abi.encodeCall(IRiscZeroReceiptVerifier.verify, (...))`

Reply:
will change soon